os/Kconfig

```diff
@@ -268,7 +268,15 @@ config PASS1_OBJECT
 	default ""
 
 endif # Build 2-pass
-endmenu # Build Setup
+
+config VERSION_BUILD_DATE
+	int "Binary Build Date"
```

Comment:
I think we'd better show the format in the prompt.
How about `Binary Build Date(YYMMDD)`?